### DISCUSSION TITLE
Failed to start container for Proxy

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#123] [Q&A]
![image](https://github.com/engageub/InternetIncome/assets/169600953/26466c27-c88e-4f7f-921f-d5002d18c3ef)
Hello community,
I got the issue to start the container, anyone get the same issue to me. Here is the logs:
Starting..
USE_PROXIES is enabled, using proxies..
v2.5.2: Pulling from xjasonlyu/tun2socks
Digest: sha256:98ef15892768ab09115e3c0d19f2c1f3ffe0f866252facb996707efaf1cac7e4
Status: Image is up to date for xjasonlyu/tun2socks:v2.5.2
docker.io/xjasonlyu/tun2socks:v2.5.2
Starting Proxy container..
"docker run" requires at least 1 argument.
See 'docker run --help'.

Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Create and run a new container from an image
internetIncome.sh: line 186: -v: command not found
Failed to start container for Proxy..Exiting..

Thanks

**Comment by engageub** [comment]
Updates were ongoing in test branch.
You may use main branch for proxies. Its very rarely updated where as test branch gets frequent updates.
However, please update the test branch now.

Thank you
